### PR TITLE
Akash Make model implementation

### PR DESCRIPTION
**Pull request by LoganWatson1400**
Akash isnt feeling great right now and asked me to put in a pull request for his work.

(Co pilot description)
This pull request introduces a new script `make_model.py` to build, train, and evaluate a Convolutional Neural Network (CNN) model using the Xception architecture. The script includes data preparation, model definition, training, evaluation, and testing.

Key changes include:

* **Model Definition and Training:**
  * Added imports for TensorFlow, Keras, and necessary layers and optimizers.
  * Defined the `build_xception_model` function to create a CNN model based on the Xception architecture with a GlobalAveragePooling2D layer, Dense layers, and Dropout for regularization.
  * Compiled the model using the Adam optimizer and binary cross-entropy loss function.
  * Trained the model on the prepared dataset and saved the trained model to a file.

* **Model Evaluation and Testing:**
  * Evaluated the model on test data and printed the test accuracy and loss.
  * Predicted on the test set and generated a classification report using `sklearn.metrics`.

### CODE REVIEW

**Review by HawkerX — DISMISSED** [review]
Everything looked fine/made sense to me. Git interpreted the model_evaluation.py a little weird, but runs/works fine (registered it as a binary file when it is actually ASCII just weird encoding stuff). Good to merge!

**Review by HawkerX — APPROVED** [review]
